ietf/templates/doc/badge/doc-badge-other.html

```diff
@@ -0,0 +1,16 @@
+{% load origin %}
+{% load static %}
+{% load ietf_filters %}
+{% load person_filters %}
+{% origin %}
+{# Non-RFC #}
+
+{% if doc.became_rfc %} 
+    <div{% if document_html %} class="alert alert-warning small"{% endif %} style="border: dashed 3px green">This is an older version of an Internet-Draft that was ultimately published as <a href="{% if document_html %}{% url 'ietf.doc.views_doc.document_html' name=doc.became_rfc.name %}{% else %}{% url 'ietf.doc.views_doc.document_main' name=doc.became_rfc.name %}{% endif %}">{{doc.became_rfc.name|prettystdname}}</a>.</div>
```

Comment:
why introduce new styling inline? Can this be moved to the css?

Reply:
I think we still have inline styling in this branch and that it's being applied in cases that aren't intended - right now the tests are failing complaining about the inline styling, and because instead of this on main:

![image](https://github.com/user-attachments/assets/834cc7f8-5f6a-436b-81af-21b704e3532e)

we're seeing this on the PR branch:

![image](https://github.com/user-attachments/assets/8aa4f614-ed36-4ab7-a988-ef440fb7b2de)

This is an active draft, so why the dashed lines, and what happened to the `(stir WG)` link?

Reply:
That `(stir WG)` link is what's passed in as `submission`